### ISSUE TITLE
ncp-restore: backups without data leave PHP down (Service Unavailable)

### ISSUE THREAD

**Issue by nachoparker** [nextcloud/nextcloudpi#178]
### What version of NextCloudPi are you using? ( eg: v0.17.2 )

Since `v0.24.3`

Only happens from ncp-web

edit: modsecurity as well

**Comment by nachoparker** [comment]
It can be brought back by rebooting, or using the following sequence

```
systemctl stop php7.0-fpm                              
systemctl stop mysqld    
systemctl start php7.0-fpm                             
systemctl start mysqld
```

**Comment by nachoparker** [comment]
fixed `v0.26.30`
